### PR TITLE
Add missing docs

### PR DESCRIPTION
**Pull request by Lulalaby**

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -2613,18 +2617,60 @@ of :class:`enum.Enum`.
...
+
+        Possible attributes for :class:`AuditLogDiff`:
+
+        - :attr:`~AuditLogDiff.name`
+        - :attr:`~AuditLogDiff.description`
+        - :attr:`~AuditLogDiff.channel_id`
+        - :attr:`~AuditLogDiff.privacy_level`
+        - :attr:`~AuditLogDiff.location`
+        - :attr:`~AuditLogDiff.status`
+        - :attr:`~AuditLogDiff.entity_type`
```
Comment (Middledot) [inline docs/api.rst:2632]:
```suggestion
        - :attr:`~AuditLogDiff.location_type`
```
It was renamed to match other names

File: docs/api.rst [hunk 2]
```diff
@@ -2613,18 +2617,60 @@ of :class:`enum.Enum`.
...
```
Comment (Middledot) [inline docs/api.rst:2652]:
```suggestion
        - :attr:`~AuditLogDiff.location_type`
```

File: docs/api.rst [hunk 3]
```diff
@@ -2613,18 +2617,60 @@ of :class:`enum.Enum`.
...
```
Comment (Middledot) [inline docs/api.rst:2672]:
```suggestion
        - :attr:`~AuditLogDiff.location_type`
```

**Review by Dorukyum — APPROVED** [review]

File: docs/api.rst [hunk 4]
```diff
@@ -2613,18 +2617,60 @@ of :class:`enum.Enum`.
...
 
+        When this is the action, the type of :attr:`~AuditLogEntry.target` is
+        the :class:`ScheduledEvent` or :class:`Object` with the ID of the thread which
+        was deleted.
+
+        Possible attributes for :class:`AuditLogDiff`:
+
+        - :attr:`~AuditLogDiff.name`
+        - :attr:`~AuditLogDiff.description`
+        - :attr:`~AuditLogDiff.channel_id`
```
Comment (Middledot) [inline docs/api.rst:2628]: I didn't notice these
```suggestion
        - :attr:`~AuditLogDiff.channel`
```

File: docs/api.rst [hunk 5]
```diff
@@ -2613,18 +2617,60 @@ of :class:`enum.Enum`.
...
```
Comment (Middledot) [inline docs/api.rst:2648]:
```suggestion
        - :attr:`~AuditLogDiff.channel`
```

File: docs/api.rst [hunk 6]
```diff
@@ -2613,18 +2617,60 @@ of :class:`enum.Enum`.
...
```
Comment (Middledot) [inline docs/api.rst:2668]:
```suggestion
        - :attr:`~AuditLogDiff.channel`
```

**Review by Middledot — APPROVED** [review]

**Review by Dorukyum — APPROVED** [review]

**Pull request merged**